### PR TITLE
Update alluxio code base to point to alluxio.io

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

**Comment by calvinjia** [comment]
@apc999 Thanks! Have you verified that all the new links are working?

File: docs/_includes/Common-Commands/download-alluxio.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ```bash
-wget http://alluxio.org/downloads/files/{{site.ALLUXIO_RELEASED_VERSION}}/alluxio-{{site.ALLUXIO_RELEASED_VERSION}}-bin.tar.gz
+wget http://alluxio.io/downloads/files/{{site.ALLUXIO_RELEASED_VERSION}}/alluxio-{{site.ALLUXIO_RELEASED_VERSION}}-bin.tar.gz
```
Comment (Xenorith) [inline docs/_includes/Common-Commands/download-alluxio.md]: alluxio.io/downloads/files/... only points people to the downloads or release notes pages. for wget, use http://downloads.alluxio.io/downloads/files/...
 
http://downloads.alluxio.io/downloads/files/{{site.ALLUXIO_RELEASED_VERSION}}/alluxio-{{site.ALLUXIO_RELEASED_VERSION}}-bin.tar.gz

File: docs/_includes/Running-Alluxio-Locally/download-Alluxio-binary.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ```bash
-wget http://alluxio.org/downloads/files/{{site.ALLUXIO_RELEASED_VERSION}}/alluxio-{{site.ALLUXIO_RELEASED_VERSION}}-bin.tar.gz
+wget http://alluxio.io/downloads/files/{{site.ALLUXIO_RELEASED_VERSION}}/alluxio-{{site.ALLUXIO_RELEASED_VERSION}}-bin.tar.gz
```
Comment (Xenorith) [inline docs/_includes/Running-Alluxio-Locally/download-Alluxio-binary.md]: wget http://downloads.alluxio.io/downloads/files/{{site.ALLUXIO_RELEASED_VERSION}}/alluxio-{{site.ALLUXIO_RELEASED_VERSION}}-bin.tar.gz

File: docs/_includes/Running-Alluxio-on-a-Cluster/download-extract-Alluxio-tar.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
 ```bash
-wget http://alluxio.org/downloads/files/{{site.ALLUXIO_RELEASED_VERSION}}/alluxio-{{site.ALLUXIO_RELEASED_VERSION}}-bin.tar.gz
+wget http://alluxio.io/downloads/files/{{site.ALLUXIO_RELEASED_VERSION}}/alluxio-{{site.ALLUXIO_RELEASED_VERSION}}-bin.tar.gz
```
Comment (Xenorith) [inline docs/_includes/Running-Alluxio-on-a-Cluster/download-extract-Alluxio-tar.md]: wget http://downloads.alluxio.io/downloads/files/{{site.ALLUXIO_RELEASED_VERSION}}/alluxio-{{site.ALLUXIO_RELEASED_VERSION}}-bin.tar.gz

File: docs/cn/FAQ.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ priority: 10
 
 1. 什么是Alluxio?
 
-   [Alluxio](http://www.alluxio.org/)（之前名为Tachyon），是一个开源的具有内存级速度的虚拟分布式存储系统，
+   [Alluxio](http://www.alluxio.io/)（之前名为Tachyon），是一个开源的具有内存级速度的虚拟分布式存储系统，
```
Comment (Xenorith) [inline docs/cn/FAQ.md]: https://

File: pom.xml [hunk 5]
```diff
@@ -18,7 +18,7 @@
   <packaging>pom</packaging>
   <name>Alluxio Parent</name>
   <description>Parent POM of Alluxio project: a Memory-Speed Virtual Distributed Storage System</description>
-  <url>http://alluxio.org/</url>
+  <url>http://alluxio.io/</url>
```
Comment (Xenorith) [inline pom.xml]: https://www.alluxio.io

Comment (apc999) [reply pom.xml]: fixed

File: docs/_includes/Common-Commands/download-alluxio.md (hunk 1 above)
Comment (apc999) [reply docs/_includes/Common-Commands/download-alluxio.md]: done

File: docs/cn/FAQ.md (hunk 4 above)
Comment (apc999) [reply docs/cn/FAQ.md]: done

File: docs/_includes/Running-Alluxio-on-a-Cluster/download-extract-Alluxio-tar.md (hunk 3 above)
Comment (apc999) [reply docs/_includes/Running-Alluxio-on-a-Cluster/download-extract-Alluxio-tar.md]: done

File: docs/_includes/Running-Alluxio-Locally/download-Alluxio-binary.md (hunk 2 above)
Comment (apc999) [reply docs/_includes/Running-Alluxio-Locally/download-Alluxio-binary.md]: done

**Comment by apc999** [comment]
@calvinjia  PTAL

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

**Pull request merged**